### PR TITLE
doc window rule `group`

### PR DESCRIPTION
**Pull request by memchr**
hyprwm/hyprland#3279

### CODE REVIEW

File: pages/Configuring/Window-Rules.md [hunk 1]
```diff
@@ -114,6 +114,25 @@ you can use `hyprctl clients`.
...
+
+## `group` window rule options
+
+- `set` \[`always`\] - Open window as a group.
+- `new` - Shorthand of `barred set`.
+- `lock` \[`always`\] - Lock the group that added this window. Use with `set` or `new` (i.e. `new lock`) to create a new locked group.
+- `barred` - Do not add the window to the focused group. By default, a window with a `group set` rule will be added to an active group if possible.
+- `deny` - Do not allow window to be toggled as or added to group (see `denywindowfromgroup` dispatcher).
+- `override` \[other options\] - Override other `group` rules, e.g. You can make all windows in a particular workspace open as a group, and use `group override barred` to make windows with specific titles open as normal windows.
+- `unset` - Clear all `group` rules.
```
Comment (vaxerski) [inline pages/Configuring/Window-Rules.md:129]: misses invade

File: pages/Configuring/Variables.md [hunk 2]
```diff
@@ -52,6 +52,8 @@ SHIFT CAPS CTRL/CONTROL ALT MOD2 MOD3 SUPER/WIN/LOGO/MOD4 MOD5
 | gaps_out | gaps between windows and monitor edges | int | 20 |
 | col.inactive_border | border color for inactive windows | gradient | 0xffffffff |
 | col.active_border | border color for the active window | gradient | 0xff444444 |
+| col.nogroup_border | inactive border color for window that cannot be added to a group (see `denywindowfromgroup` dispatcher) | 0xffffaaff |
```
Comment (vaxerski) [inline pages/Configuring/Variables.md:55]: missed type in both

File: pages/Configuring/Window-Rules.md (hunk 1 above)
Comment (memchr) [reply pages/Configuring/Window-Rules.md:129]: it's a last minute thing.

**Review by vaxerski — APPROVED** [review]
m

**Pull request merged**